common/automate-custom-button.adoc

```diff
@@ -20,16 +20,30 @@ image:../images/1862.png[image] (*Add a new Button*).
 ====
 If image:../images/1862.png[image] (*Add a new Button*) is not available, that means you have not created a button group for that object. To continue, create a button group first. See <<create-custom-button-group>>
 ====
-. In *Action*, type in a *Button Text* and *Button Hover Text*, and select the *Button Image* you want to use.
 
-. Select a *Dialog* if applicable.
-
-. In *Object Details*, select *Request* from the `/System/Process/` dropdown. By default, the message is `create`. Do not change it.
+. Under the *Options* tab:
+.. Select the *Button* type from the list. 
+.. Type in button *Text* and *Hover Text*, and select the *Icon* and *Icon Color* to use.
```

Comment:
@cbudz I realize this is probably old text but would you mind changing the procedure steps to use "Enter" instead of "Type in" per convention. This applies to all steps where we are providing an input using a keyboard.